pkg/nsm/objectstore/objectstore.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objectstore
+
+import (
+	"sync"
+
+	"github.com/ligato/networkservicemesh/netmesh/model/netmesh"
+)
+
+// meta is used as a key for each NetworkService object
+type meta struct {
+	name      string
+	namespace string
+}
+
+// NetworkServicesStore map stores all discovered Network Service Object
```

Comment:
Do we also want to cache channels and endpoints?

Reply:
Eventually yes, at this point I just wanted to be able to list `fake` NetworkServices and reply to client's `Discovery` call. If PR will be relatively small, I will add them, but I suspect  with them it might be too big for a good review.

Reply:
This makes sense to me, we can add them in a followup. Thanks!